waiter/test/waiter/token_test.clj

```diff
@@ -814,19 +851,24 @@
             (let [body-map (-> body str json/read-str)]
               (doseq [key sd/service-parameter-keys]
                 (is (= (get service-description key) (get body-map key))))
-              (doseq [key (disj sd/user-metadata-keys "stale-timeout-mins")]
-                (is (= (get service-description key) (get body-map key))))
+              (doseq [key (disj sd/user-metadata-keys "cluster" "stale-timeout-mins")]
```

Comment:
adding `cluster` here seems unnecessary, because the sd/user-metadata-keys do not contain `cluster`. `cluster` should remain a system metadata key.

Reply:
The cluster is showing up because it is stored in the `kv-store` from line #521 above.

Reply:
Actually, you're right. This should not be showing up in the response. I may have added this incorrectly. Checking...

Reply:
Removed unnecessary operations with the cluster key from the tests.